rabbit-escape-engine/src/rabbitescape/engine/BehaviourTools.java

```diff
@@ -63,6 +63,11 @@ public boolean rabbitIsFalling()
         }
     }
 
+    public boolean rabbitIsClimbing()
+    {
+        return rabbit.state.toString().contains( "CLIMB" );
+    }
+    
```

Comment:
I'd prefer a switch on enum values here. But I think we are seeing more and more cases where we need more structure than a flat list. We should start by removing the redundant left/right repetition.